Agencia del ISBN.js

```diff
@@ -46,31 +46,69 @@ function doWeb(doc, url) {
 }
 		
 		
-	function scrape (doc, url){	
-		var data = new Object();
-		var rows = doc.evaluate('//div[@class="fichaISBN"]/table/tbody/tr', doc, null, XPathResult.ANY_TYPE, null);
-		var next_row;
-		while (next_row = rows.iterateNext()) {
-			var heading = doc.evaluate('./th', next_row, null, XPathResult.ANY_TYPE, null).iterateNext().textContent;
-			var value = doc.evaluate('./td', next_row, null, XPathResult.ANY_TYPE, null).iterateNext().textContent;
-			data[heading.replace(/\W/g, "")] = value;
-		}
-		var isbn = Zotero.Utilities.trimInternal(doc.evaluate('//span[@class="cabTitulo"]/strong', doc, null, XPathResult.ANY_TYPE, null).iterateNext().textContent);
-		var item = new Zotero.Item("book");
-		item.ISBN = isbn;
-		item.title = Zotero.Utilities.trimInternal(data['Ttulo']);
-		item.title= item.title.replace(/\s+:/, ":");
-		author = data['Autores'];
-		if (author) {
-			var authors = author.match(/\b.*,\s+\w+[^([]/g);
-			for each (aut in authors) {
-				item.creators.push(Zotero.Utilities.cleanAuthor(Zotero.Utilities.trimInternal(aut), "author", true));
-			}
+function scrape (doc, url){	
+	var data = new Object();
+	var rows = doc.evaluate('//div[@class="fichaISBN"]/table/tbody/tr', doc, null, XPathResult.ANY_TYPE, null);
+	var next_row;
+	while (next_row = rows.iterateNext()) {
+		var heading = doc.evaluate('./th', next_row, null, XPathResult.ANY_TYPE, null).iterateNext().textContent;
+		var value = doc.evaluate('./td', next_row, null, XPathResult.ANY_TYPE, null).iterateNext().textContent;
+		data[heading.replace(/\W/g, "")] = value;
+	}
+	var isbn = Zotero.Utilities.trimInternal(doc.evaluate('//span[@class="cabTitulo"]/strong', doc, null, XPathResult.ANY_TYPE, null).iterateNext().textContent);
+	var item = new Zotero.Item("book");
+	item.ISBN = isbn;
+	item.title = Zotero.Utilities.trimInternal(data['Ttulo']);
+	item.title= item.title.replace(/\s+:/, ":");
+	author = data['Autores'];
+	if (author) {
+		var authors = author.match(/\b.*,\s+\w+[^([]/g);
+		for (var i=0; i<authors.length; i++) {
+			var aut = Zotero.Utilities.trimInternal(authors[i]);
+			item.creators.push(Zotero.Utilities.cleanAuthor(aut, "author", true));
 		}
-		if (data['Publicacin']) item.publisher = Zotero.Utilities.trimInternal(data['Publicacin']);
-		if (data['FechaEdicin']) item.date = Zotero.Utilities.trimInternal(data['FechaEdicin']);
-		item.complete();
 	}
+	if (data['Publicacin']) item.publisher = Zotero.Utilities.trimInternal(data['Publicacin']);
+	if (data['FechaEdicin']) item.date = Zotero.Utilities.trimInternal(data['FechaEdicin']);
+	item.complete();
+}
+
```

Comment:
I sometimes had problems to read the code because the whitespacing was wrong and therefore I tried to improve a little. Is this in general a bad idea or are you more concenrned about the git history?